code/datums/elements/undertile.dm

```diff
@@ -47,6 +49,8 @@
 			T.overlays -= tile_overlay
 		if(use_alpha)
 			source.alpha = 255
+		if(use_anchor)
+			source.anchor = FALSE
```

Comment:
This can unanchor things that must be anchored, 
Be sure that dont add  use_anchor = TRUE to things that must be anchored. 
or save state on hiding,

or sec anchor initial but this can dont work in some cases

Reply:
If I start storing or using initial, it’ll get fucked at some point. But yeah I commented about how this should only be used for simple items like satchels.

You’re right though, I don’t like it either but it’s better than adding snowflake signals for stuff like satchels. Which is ironically what people will need to do if they got some more advanced stuff that needs special anchoring

Reply:
Yes